.github/workflows/github-ci.yml

```
@@ -0,0 +1,126 @@
+        run: npm install
+      - name: Build the project
+        run: npm run build
+
+  i18n:
+    name: Verify Translations
+    runs-on: ubuntu-latest
+    needs: [ build ]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)